internal/beef/beef_test.go

```diff
@@ -0,0 +1,284 @@
+package beef
+
+import (
+	"encoding/hex"
+	"errors"
+	"testing"
+
+	"github.com/libsv/go-bc"
+	"github.com/libsv/go-bt/v2"
+	"github.com/libsv/go-bt/v2/bscript"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCheckBeefFormat(t *testing.T) {
```

Comment:
nit: You could also make this test table based like the others

Reply:
Done :)